installation/synology.md

```diff
@@ -37,23 +47,12 @@ If your NAS cannot connect to the internet, the installer will tell you to downl
 
 ## Configuration
 
-The configuration files are located at:
+The location of the configuration files varies depending if a shared folder named public exists. The public folder is created automatically after your Synology NAS is first set up for certain models. The administrator can also create a public shared folder if desired. Synology does not document what models will automatically create a public folder. See [DiskStation Manager - Shared Folder](https://www.synology.com/en-us/knowledgebase/DSM/help/DSM/AdminCenter/file_share_desc)
 
-```conf
-/volume1/@appstore/openHAB2/userdata/config
-```
+If a public shared folder exists, the OpenHAB install script will place configuration files are located at:
 
-## Tested Devices
+`/volume1/public/openHAB2/conf/`
 
-| DiskStation | Package version |
-|-------------|-----------------|
-| DS209       | [2.0.0-beta3](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0-beta3) |
-| DS216+II    | [2.0.0-beta3](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0-beta3) |
-| DS216+II    | [2.0.0.009-SNAPSHOT-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.009-SNAPSHOT-DSM6) |
-| DS216play   | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS413j      | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS916+      | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
-| DS214play   | [2.1.0.004-SNAPSHOT-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.1.0.004-SNAPSHOT-DSM6) |
-| DS1815+     | [2.0.0.001-DSM6](https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.001-DSM6) |
```

Comment:
The list is constantly changing and we are currently developing a package for all DiskStation models.